### PR TITLE
Fix AnalysisConfigurer example in Hibernate Search documentation

### PR DESCRIPTION
**Pull request by viniciusfcf**
Fix configure parameter Classs. Changing ElasticsearchAnalysisDefinitionContainerContext to ElasticsearchAnalysisConfigurationContext

### CODE REVIEW

File: docs/src/main/asciidoc/hibernate-search-elasticsearch.adoc [hunk 1]
```diff
@@ -476,8 +476,8 @@ In our existing `LibraryResource`, we just need to inject the following methods
     @GET
     @Path("author/search") // <3>
     @Transactional
-    public List<Author> searchAuthors(@QueryParam String pattern, // <4>
-            @QueryParam Optional<Integer> size) {
+    public List<Author> searchAuthors(@QueryParam("pattern") String pattern, // <4>
+            @QueryParam("size") Optional<Integer> size) {
```
Comment (gsmet) [inline docs/src/main/asciidoc/hibernate-search-elasticsearch.adoc]: This change shouldn't be needed, see the note: you should use `org.jboss.resteasy.annotations.jaxrs.QueryParam` for the import.

**Review by gsmet — CHANGES_REQUESTED** [review]
The first commit is legit, thanks for the fix. But the second one is not. Could you remove the second commit?

Thanks!

File: docs/src/main/asciidoc/hibernate-search-elasticsearch.adoc (hunk 1 above)
Comment (viniciusfcf) [reply docs/src/main/asciidoc/hibernate-search-elasticsearch.adoc]: Ok, sorry.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Cool! Thanks for taking the time to create a PR, very much appreciated.

**Pull request merged**